### PR TITLE
Add support for Kami65 PCB 

### PR DESCRIPTION
**Pull request by MaiTheSan**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Per the PR checklist,

> * PRs should contain the smallest amount of modifications required for a single change to the codebase
>    * multiple keyboards at the same time is not acceptable

**Comment by MaiTheSan** [comment]
> Per the PR checklist,
> 
> > * PRs should contain the smallest amount of modifications required for a single change to the codebase
> >   
> >   * multiple keyboards at the same time is not acceptable

got it

**Comment by dunk2k** [comment]
please:
1. `rename keyboards/chickenman/kami65/info.json` to `keyboards/chickenman/kami65/keyboard.json`
2. delete `keyboards/chickenman/kami65/rules.mk`

As so this PR conforms to the recent changes regarding build markers.

File: keyboards/chickenman/kami65/halconf.h
Comment (drashna) [inline keyboards/chickenman/kami65/halconf.h:1]: This file isn't doing anything, and should be removed.

File: keyboards/chickenman/kami65/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Kami65
+
+![Kami65](https://i.imgur.com/SwXfDcr.png)
+
+A 65% keyboard.
+
+* Keyboard Maintainer: [Mai The San](https://github.com/maithesan)
+* Hardware Supported: Kami65 Keyboard
+* Hardware Availability: UCSD Discord
```
Comment (drashna) [inline keyboards/chickenman/kami65/readme.md:9]: Do you have a more specific link, or a link to the design?

**Comment by tzarc** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

File: keyboards/chickenman/kami65/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x00A1",
+        "vid": "0xC41C"
+    },
+    "ws2812": {
+        "pin": "B15"
+    },
```
Comment (dunk2k) [inline keyboards/chickenman/kami65/keyboard.json:49]:
```suggestion
    },
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_65_ansi_blocker_split_bs"
    },
```

File: keyboards/chickenman/kami65/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │PgU│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │  Enter │PgD│
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+     * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift│ ↑ │End│
+     * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+     * │Ctrl│GUI │Alt │                        │ Alt│  Fn│ │ ← │ ↓ │ → │
+     * └────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_65_ansi_blocker_split_bs(
```
Comment (dunk2k) [inline keyboards/chickenman/kami65/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_all(
```

File: keyboards/chickenman/kami65/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+     * │     │   │   │   │   │   │Ins│   │   │   │   │PSc│Scr│Pause│   │
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+     * │      │   │   │   │   │   │   │   │   │   │   │   │        │   │
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+     * │        │   │   │   │   │   │   │Mut│Vl-│Vl+│   │      │   │   │
+     * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+     * │    │    │    │                        │    │    │ │   │   │   │
+     * └────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
+     */
+    [1] = LAYOUT_65_ansi_blocker_split_bs(
```
Comment (dunk2k) [inline keyboards/chickenman/kami65/keymaps/default/keymap.c:42]:
```suggestion
    [1] = LAYOUT_all(
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
conforms to PR guidance regarding PCBs with more than one physical layout

**Review by drashna — APPROVED** [review]

**Pull request closed without merging**